web/src/components/organisms/Asset/convertAsset.ts

```diff
@@ -13,5 +18,16 @@ export const convertAsset = (GQLAsset: GQLAsset | undefined): Asset | undefined
     projectId: GQLAsset.projectId,
     size: GQLAsset.size,
     url: GQLAsset.url,
+    threadId: GQLAsset.thread?.id ?? "",
+    comments: GQLAsset.thread?.comments?.map(comment => convertComment(comment)) ?? [],
+  };
+};
+
+export const convertComment = (GQLComment: GQLComment): Comment => {
```

Comment:
For now its fine, but notice we have I think a few convert functions now for converting to/from GQL types. 
It'd be nice to consolidate them in a single file (like we do in reearth-web).

Reply:
Ok, this makes sense. I will do it in a separate PR.